<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JudgementQ</class>
 <widget class="QWidget" name="JudgementQ">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>618</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>618</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>618</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>正在答题</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 170, 173);</string>
  </property>
  <widget class="QPushButton" name="next_btn">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>220</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>28</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 224, 102);</string>
   </property>
   <property name="text">
    <string>⇨</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="question_text">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>700</width>
     <height>300</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="previous_btn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>28</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 224, 102);</string>
   </property>
   <property name="text">
    <string>⇦</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </widget>
  <widget class="QPushButton" name="True_btn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>430</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>√(T)</string>
   </property>
   <property name="shortcut">
    <string>T</string>
   </property>
  </widget>
  <widget class="QPushButton" name="False_btn">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>430</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>×(F)</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
  </widget>
  <widget class="QLabel" name="index_lbl">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>580</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(112, 193, 179);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
